parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -868,6 +868,24 @@ providers:
     apiVersion: credentialprovider.kubelet.k8s.io/v1
     args:
       - /etc/kubernetes/azure.json
+EOF
+    elif [[ -n "${BOOTSTRAP_PROFILE_CONTAINER_REGISTRY_SERVER}" ]]; then
```

Comment:
we don't have network-isolated cluster for sov cloud? because AKS_CUSTOM_CLOUD_CONTAINER_REGISTRY_DNS_SUFFIX and BOOTSTRAP_PROFILE_CONTAINER_REGISTRY_SERVER use different logics

Reply:
network-isolated cluster supports sov cloud. The bollow suffix is for sov cloud.
```
      - "*.azurecr.cn"
      - "*.azurecr.de"
      - "*.azurecr.us"
 ```


But `AKS_CUSTOM_CLOUD_CONTAINER_REGISTRY_DNS_SUFFIX` is for custom cloud. I believe custom cloud is not sov cloud. network-isolated cluster doesn't support custom cloud.